MapboxCoreNavigation/RouteController.swift

```diff
@@ -54,7 +54,15 @@ public protocol RouteControllerDelegate: class {
 }
 
 /**
- A `RouteController` tracks the user’s progress along a route, posting notifications as the user reaches significant points along the route. On every location update, the route controller evaluates the user’s location, determining whether the user remains on the route. If not, the route controller calculates a new route.
+ A `RouteController` tracks the user’s progress along a route, posting 
+ notifications as the user reaches significant points along the route. On 
+ every location update, the route controller evaluates the user’s location, 
+ determining whether the user remains on the route. If not, the route 
+ controller calculates a new route.
+ 
+ `RouteController` is responsible for the core navigation logic whereas 
+ `NavigationViewController` is responsible for displaying a default drop-in 
+ navigation UI.
```

Comment:
I prefer 80-column hard wrapping in Objective-C libraries but am ambivalent about it in Swift. We aren’t sticking to an 80-column code style in the Swift libraries. Also, when you jump to the definition of any of these symbols in Xcode (whether from Objective-C or Swift), the Swift compiler will automatically reflow this text, losing the 80-column hard wrapping. (That might not be the case with CocoaPods projects, since the workspace would include the source files in that case.)